data/migrations/25.lua

```diff
@@ -1,3 +1,6 @@
 function onUpdateDatabase()
-	return false
-end
+	print("> Updating database to version 26 (Better premium time handling)")
+	db.query("ALTER TABLE `accounts` DROP COLUMN `lastday`")
+	db.query("ALTER TABLE `accounts` CHANGE COLUMN `premdays` `premend` int(11) NOT NULL DEFAULT 0");
```

Comment:
```db.query("UPDATE `accounts` SET `premend`= premend * 86400")```
So it updates the values correctly and fixes what @slawkens and @Znote suggested.